deploy/registration-service.yaml

```diff
@@ -257,3 +269,7 @@ parameters:
     value: ''
   - name: TWILIO_FROM_NUMBER
     value: ''
+  - name: WOOPRA_DOMAIN
+    value: ''
+  - name: SEGMENT_WRITE_KEY
+    value: ''
```

Comment:
This end of the file doesn't correspond to the one in host-operator repo - could you please fix that? https://github.com/codeready-toolchain/host-operator/pull/326/files#diff-43fc4f9268f9dd2811ffed0c88d92da522f5b2c0cb3e2643cf253243e2025276R275

Reply:
fixed here: 47b068cff45b7c35ac2543646c015ce4c2bb1bcf